.gitignore

```diff
@@ -16,3 +16,7 @@ rustup/*
 
 # clangd
 compile_commands.json
+
+# cmps (https://github.com/leonmatthes/cmps)
+.cmps/
```

Comment:
is there something in this commit that relates to this change?

Reply:
No, but I've added license-templates to cmps now :partying_face: 
Could split this into a separate PR if necessary, thought it a bit of a small change for that though.